docs/iris/src/whatsnew/1.10.rst

```diff
@@ -0,0 +1,171 @@
+What's New in Iris 1.10
+***********************
+
+:Release: 1.10
+:Date: 2016-08-08
+
+This document explains the new/changed features of Iris in version 1.10
+(:doc:`View all changes <index>`.)
+
+Iris 1.10 Features
+==================
+.. _iris_grib_added:
+
+* Support has now been added for the
+  `iris_grib <https://github.com/SciTools/iris-grib>`_ package, which
+  provides GRIB format support in an optional package, separate from Iris.
+
+  * If ``iris_grib`` is available, it will always be used in place of the older
+    iris module :mod:`iris.fileformats.grib`.
+
+  * The capabilities of ``iris_grib`` are essentially the same as the existing
+    :mod:`iris.fileformats.grib` when used with ``iris.FUTURE.strict_grib_load=True``,
+    with only small detail differences.
+
+  * The old :mod:`iris.fileformats.grib` module is now deprecated and may shortly be
+    removed.
+
+    * If you are already using the recommended :data:`iris.FUTURE` setting
+      ``iris.FUTURE.strict_grib_load=True`` this should not cause problems, as
+      the new package is all-but identical.
+
+      * However, the option ``iris.FUTURE.strict_grib_load`` is itself now
+        deprecated, so you should remove code that sets it.
+
+    * If, however, your code is still using the older "non-strict" grib
+      loading, then you may need to make code changes.
+
+      * In particular, the ``field`` object passed to load callbacks is
+        different.
+        See :class:`iris.fileformats.grib.message.GribMessage` (the
+        ``iris_grib.message.GribMessage`` class is the same as this, for now).
+
+  * Please exercise your code with the new iris_grib module, and let us know of
+    any problems you uncover, such as files that will no longer load with the
+    new implementation.
+
+* :meth:`iris.experimental.regrid.PointInCell.regridder` now works across coordinate systems, including non latlon systems. Additionally, the requirement that the source data X and Y coordinates be 2D has been removed. NB: some aspects of this change are backwards incompatible.
+* Plotting non-Gregorian calendars is now supported. This adds `nc_time_axis <https://github.com/SciTools/nc-time-axis>`_ as a dependency.
+* Promoting a scalar coordinate to a dimension coordinate with :func:`iris.util.new_axis` no longer loads deferred data.
+* The parsing functionality for Cell Methods from netCDF files is available as part of the :mod:`iris.fileformats.netcdf` module as :func:`iris.fileformats.netcdf.parse_cell_methods`.
+* Support for the NameIII Version 2 file format has been added.
+* The Mercator projection is now supported with defaults for some parameters (``false easting``, ``false northing`` and ``scale factor at projection origin``).
```

Comment:
> "..now supported with defaults for some parameters.."

Best to call these " _projection_ parameters ", as they are netcdf attributes and ***not*** (Python) parameters of the projection classes.  Even then, it could be clearer... 

? How about ?:
```
* Loading netcdf data with Mercator and Stereographic projection types now supports extra attributes matching
  the default settings :  (``false_easting``, ``false_northing`` and ``scale_factor_at_projection_origin``)
```

**Note** I've also put underscores into all those names :  This is the correct naming according to Netcdf-CF.